packages/api/src/external/carequality/command/cq-directory/search-cq-directory.ts

```diff
@@ -75,9 +75,13 @@ export async function searchCQDirectoriesByRadius({
           ],
         ],
       },
-      where:
-        Sequelize.literal(`earth_box(ll_to_earth (${coord.lat}, ${coord.lon}), ${radiusInMeters}) @> point
-        AND earth_distance(ll_to_earth (${coord.lat}, ${coord.lon}), point) < ${radiusInMeters}`),
+      where: {
+        [Op.and]: [
+          Sequelize.literal(`earth_box(ll_to_earth (${coord.lat}, ${coord.lon}), ${radiusInMeters}) @> point
+      AND earth_distance(ll_to_earth (${coord.lat}, ${coord.lon}), point) < ${radiusInMeters}`),
+          { urlXCPD: { [Op.ne]: "" } },
```

Comment:
Can't it be `null`?

Reply:
Can we add a parameter to indicate whether to filter by the presence of `urlXCPD`? It can defs be enabled by default.

Just because it's not obvious from the function name and docs that it only returns items with that field present.

Reply:
can't use `null`. Maybe can use `undefined` instead. 
And yes, second comment def makes sense!

Reply:
Ended up using a string literal for the where clause. In that case, using null is 👍